packages/editor-ui/src/views/TestDefinition/TestDefinitionRunDetailView.vue

```diff
@@ -28,6 +28,65 @@ const filteredTestCases = computed(() => {
 	return testCases.value;
 });
 
+const getErrorTooltipLinkRoute = computed(() => (row: TestCaseExecutionRecord) => {
+	if (row.errorCode === 'FAILED_TO_EXECUTE_EVALUATION_WORKFLOW') {
+		return {
+			name: VIEWS.EXECUTION_PREVIEW,
+			params: {
+				name: test.value?.evaluationWorkflowId,
+				executionId: row.evaluationExecutionId,
+			},
+		};
+	} else if (row.errorCode === 'MOCKED_NODE_DOES_NOT_EXIST') {
+		return {
+			name: VIEWS.TEST_DEFINITION_EDIT,
+			params: {
+				testId: testId.value,
+			},
+		};
+	} else if (row.errorCode === 'FAILED_TO_EXECUTE_WORKFLOW') {
+		return {
+			name: VIEWS.EXECUTION_PREVIEW,
+			params: {
+				name: test.value?.workflowId,
+				executionId: row.executionId,
+			},
+		};
+	} else if (row.errorCode === 'TRIGGER_NO_LONGER_EXISTS') {
+		return {
+			name: VIEWS.EXECUTION_PREVIEW,
+			params: {
+				name: test.value?.workflowId,
+				executionId: row.pastExecutionId,
+			},
+		};
+	} else if (row.errorCode === 'METRICS_MISSING') {
+		return {
+			name: VIEWS.TEST_DEFINITION_EDIT,
+			params: {
+				testId: testId.value,
+			},
+		};
+	} else if (row.errorCode === 'UNKNOWN_METRICS') {
+		return {
+			name: VIEWS.TEST_DEFINITION_EDIT,
+			params: {
+				testId: testId.value,
+			},
+		};
+	} else if (row.errorCode === 'INVALID_METRICS') {
+		return {
+			name: VIEWS.EXECUTION_PREVIEW,
+			params: {
+				name: test.value?.evaluationWorkflowId,
+				executionId: row.evaluationExecutionId,
+			},
+		};
+	}
+
+	return undefined;
+});
+
```

Comment:
IMO this would fit better in TableCell, or ideally new TableStatusCell component as we already have some error mapping there(`errorTooltipMap`). It also shouldn't be a computed prop, since you're returning a method anyway